examples/clone/clone.bal

```diff
@@ -35,29 +35,29 @@ public function main() {
         address: address
     };
 
-    // Assign the `person` value to an `any` typed variable called `anyValue`.
-    // The type of the value held by `anyValue` is determined at runtime.
+    // Assigns the `person` value to an `any`-typed variable called `anyValue`.
+    // The type of the value held by `anyValue` is determined during runtime.
```

Comment:
BTW shouldn't it be "run time" (two words)? "runtime" (single word) is used to refer to the execution environment.

Reply:
Not quite sure.

Spec says runtime, also off Wikipedia "In computer science, run time, runtime or execution time is the time during which a program is running (executing), in contrast to other program lifecycle phases such as compile time, link time and load time."

Reply:
"runtime" is the noun and "run-time" is the adjective. It is more accurate to say "during the runtime."

Reply:
Regarding "during the" vs "at", while both seem to be used, "at" seems to be used more commonly compared to "during the" (based on a simple Google search at least :D ).

Even in these cases, we are talking about particular points at runtime, where the type is determined. Shall we stick to the spec terms, WDYT? @anupama-pathirage, @pubudu91 thoughts?

Reply:
Mostly, "at" is used to refer to a venue (i.e., where something happens). Thus, saying this happens during the runtime (i.e., referring to the time in between the start and end points of the runtime) is more meaningful. @anupama-pathirage and @pubudu91, will wait for your comments also.

Reply:
Hmm.. I'm not sure. To me, personally, "runtime" sounds weird/wrong since I've always associated it with the execution environment, and not at-the-time-of-execution.

Reply:
@pubudu91 - Please comment on the use of "at runtime" or "during runtime".

Reply:
Oh sorry. Both sound ok to me. Went through the usages of "at runtime" in the spec, and while in some places it sounds ok, in some cases during runtime sounds more appropriate.

Reply:
Thanks for the input, Pubudu! @anupama-pathirage WDYT?

Reply:
IMO "at runtime" sounds ok. My understanding is "runtime" is the environment where it get runs. So here "at runtime" refers where 'anyValue' is determined exactly. But honestly this is my personal opinion and I couldn't find official technical documentation to prove this:) 

But found below, which exactly matches with my understanding on "run time", "run-time" and "runtime".

https://english.stackexchange.com/questions/67013/runtime-run-time-and-run-time/197688#197688

Reply:
+1 Thanks Anupama.

Reply:
Thank you everyone. :D

Reply:
```suggestion
    // The type of the value held by `anyValue` is determined at runtime.
```